experimental/packages/opentelemetry-sdk-metrics-base/test/Meter.test.ts

```diff
@@ -295,6 +295,11 @@ describe('Meter', () => {
         const counter = meter.createCounter('name with invalid characters^&*(');
         assert.ok(counter instanceof api.NoopMetric);
       });
+
+      it('should return no op metric if name exceeded length of 63', () => {
+        const counter = meter.createCounter('name_with_length_of_64_so_this_is_soooooooooooooooooooooooo_long');
```

Comment:
for better readibilitty
```suggestion
        const counter = meter.createCounter('a'.repeat(63));
        assert.ok(counter instanceof CounterMetric);
        const counter = meter.createCounter('a'.repeat(64));
        assert.ok(counter instanceof api.NoopMetric);
```